python/paddle/fluid/tests/unittests/test_logaddexp.py

```diff
@@ -0,0 +1,73 @@
+#  Copyright (c) 2020 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+
+import paddle
+
+
+def ref_logaddexp_old(x, y):
+    y = np.broadcast_to(y, x.shape)
+    out = np.log(1 + np.exp(np.minimum(x, y) - np.maximum(x, y))) + np.maximum(
```

Comment:
it is better to use `np.log1p` and `np.absolute`?

Reply:
done